### ISSUE TITLE
improve testing and verbose output for env tests

### ISSUE THREAD

**Issue by jscook2345** [apptainer/singularity#3622]
## Version of Singularity:

master

###

https://github.com/sylabs/singularity/blob/master/e2e/env/env.go

Right now, it prints everything in the environment (printenv) and checks if a specific string is present. That leads to massive, hard to follow verbose output when the test fails.

I would break this up into 3 functions (one for each environment variable) and make sure their output is a little cleaner when they fail. I have ideas for this!

I would also make sure to use the new testing library that @mem setup under `github.com/sylabs/singularity/internal/pkg/test/exec`.

**Comment by dtrudg** [comment]
These are now using `e2e.ExactMatch` and echo-ing specific env vars.
